[2024-01-19T10:40:19.651+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: big_data_project_dag.store_on_hdfs scheduled__2024-01-18T00:00:00+00:00 [queued]>
[2024-01-19T10:40:19.663+0100] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: big_data_project_dag.store_on_hdfs scheduled__2024-01-18T00:00:00+00:00 [queued]>
[2024-01-19T10:40:19.664+0100] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-19T10:40:19.700+0100] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): store_on_hdfs> on 2024-01-18 00:00:00+00:00
[2024-01-19T10:40:19.710+0100] {standard_task_runner.py:60} INFO - Started process 5907 to run task
[2024-01-19T10:40:19.715+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'big_data_project_dag', 'store_on_hdfs', 'scheduled__2024-01-18T00:00:00+00:00', '--job-id', '97', '--raw', '--subdir', 'DAGS_FOLDER/CoinMarketCap.py', '--cfg-path', '/tmp/tmpjkhegx86']
[2024-01-19T10:40:19.719+0100] {standard_task_runner.py:88} INFO - Job 97: Subtask store_on_hdfs
[2024-01-19T10:40:19.781+0100] {task_command.py:423} INFO - Running <TaskInstance: big_data_project_dag.store_on_hdfs scheduled__2024-01-18T00:00:00+00:00 [running]> on host ubuntu.ubuntu.virtualbox.org
[2024-01-19T10:40:19.956+0100] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jules' AIRFLOW_CTX_DAG_ID='big_data_project_dag' AIRFLOW_CTX_TASK_ID='store_on_hdfs' AIRFLOW_CTX_EXECUTION_DATE='2024-01-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-18T00:00:00+00:00'
[2024-01-19T10:40:19.975+0100] {client.py:192} INFO - Instantiated <InsecureClient(url='http://localhost:9870')>.
[2024-01-19T10:40:19.976+0100] {client.py:496} INFO - Writing to '/your/hdfs/path/CryptoList_10-40-19.json'.
[2024-01-19T10:40:22.410+0100] {logging_mixin.py:188} INFO - Step 4: Data has been stored on HDFS at /your/hdfs/path/CryptoList_10-40-19.json
[2024-01-19T10:40:22.411+0100] {python.py:201} INFO - Done. Returned value was: None
[2024-01-19T10:40:22.422+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=big_data_project_dag, task_id=store_on_hdfs, execution_date=20240118T000000, start_date=20240119T094019, end_date=20240119T094022
[2024-01-19T10:40:22.457+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-19T10:40:22.469+0100] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
